packages/widget/src/components/SendToWallet/SendToWallet.tsx

```diff
@@ -31,6 +31,9 @@ export const SendToWallet: React.FC<BoxProps> = forwardRef((props, ref) => {
     rules: {
       required:
         requiredToAddress && (t('error.title.walletAddressRequired') as string),
+      onChange: (e) => {
+        setValue(FormKey.ToAddress, e.target.value.trim());
```

Comment:
This looks solid, but I have an very peculiar edge case where the user might paste an ens with a space in middle. To solve for this, we can use a regex to remove all spaces, as `.trim()` also uses a regex afaik internally.

This should work the best, imo
`string.replace(/\s/g, '')`

Reply:
Space in the middle? 👀

Reply:
haha, yeah, not super peculiar edge case, only possible if the user "*wants-wants*" to do it.

Reply:
I mean, are spaces allowed in ENS?

Reply:
> I mean, are spaces allowed in ENS?

Don't believe they are